templates/free-text.json

```diff
@@ -3,17 +3,13 @@
 	"subject":"${subject}",
```

Comment:
Just to check, do we need to prepend "Diakonie" here as well?

Reply:
I checked in UserService, it is the fixed value "Deletion workflow errors", so no need to adapt.